generators/client/templates/angular/src/main/webapp/app/home/home.component.ts.ejs

```diff
@@ -16,10 +16,8 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-import { Component, OnInit<%_ if (authenticationType !== 'oauth2') { _%>, OnDestroy <%_ } _%> } from '@angular/core';
-<%_ if (authenticationType !== 'oauth2') { _%>
-import { Subscription } from 'rxjs';
-<%_ } _%>
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { takeUntil } from 'rxjs/operators';
```

Comment:
Why polluting `oauth2` with unnecessary code? I think that for `oauth2` all this code is unnecessary.